src/naf-dialog-adapter.js

```diff
@@ -202,36 +224,91 @@ export default class DialogAdapter {
    * https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/setConfiguration
```

Comment:
That's what I understand and what I saw during testing. When using TURN we need to update the ICE servers in case the credentials have expired otherwise we are good with just restarting ICE in both sides. As yous y say, if we are not using proxy candidates we can just restart ICE without updating the servers. 

We can also check here if we are we are using a non proxy candidate to avoid an ICE server update as even when forcing TURN we still add a STUN server. I was assuming here that if TURN is forced is because the client is having ICE issues using STUN.

Reply:
Even when a candidate selected via STUN fails the STUN server itself should not need to change so I think if we can fix the credentials issue we should not ever need to change the ICE servers (except in the case of a dialog server transfer